ssz-rs/Cargo.toml

```
@@ -16,18 +16,24 @@ exclude = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["serde"]
+default = ["serde", "std"]
+std = [
+    "bitvec/std",
+    "sha2/std",
+    "num-bigint/std",
+    "serde/std"
+]
```

unless there is some collision, it seems better to just pull in the `default` feature set for each crate

```suggestion
std = [
    "bitvec/default",
    "sha2/default",
    "lazy_static/default",
    "num-bigint/default",
]
```

I also think we can drop `serde` here? may have to try to build w/o it and see what happens

---

Unfortunately `lazy-static` doesn't have a `default` feature: https://github.com/rust-lang-nursery/lazy-static.rs/blob/master/Cargo.toml

But the rest works nicely, I could drop `serde` here, as you suggested.

---

ah then I think we can just do `"lazy_static"` and then keep it "default-features false" and use the `spin_no_std` feature for our `no_std` setting

---

it turns out that `lazy_static` is not easy to integrate in an additive way, so I just dropped it and added a build script to generate the required data at compile time, see #42 for more info